Scripts/package.py

```diff
@@ -122,25 +122,30 @@ def build(self):
 
         if path.exists(self.buildDirectory):
             shutil.rmtree(self.buildDirectory)
+        env = dict(os.environ)
+        env["RUNTIME_IDENTIFIER"] = self.target
         run(["make", "--quiet", "clean"])
         run(["dotnet", "publish", path.join(SOURCE_DIRECTORY, "DafnyLanguageServer", "DafnyLanguageServer.csproj"),
             "--nologo",
             "-f", "net6.0",
             "-o", self.buildDirectory,
             "-r", self.target,
-            "-c", "Release"])
+            "--sc",
```

Comment:
I had to figure out manually that this option means "self-contained". Since this is our script, can you use the non abbreviate version?